### DISCUSSION TITLE
El formato de la entrega del pokedex

### DISCUSSION THREAD

**Discussion by TipiQss** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#494] [Q&A]
Hola habia tenido unos problema y no habia podido subir el pokedex aun no esta el formato para entregarlo es que no lo encuento

**Comment by OscarPedroza** [comment]
Si aun esta disponible compañero: [https://forms.office.com/r/DEqyEJMXcz](https://forms.office.com/r/DEqyEJMXcz)
